ocs_ci/ocs/scale_lib.py

```diff
@@ -878,14 +888,67 @@ def check_and_add_enough_worker(worker_count):
                 )
             return True
 
+        # Add enough worker for vmware
+        elif (
+            config.ENV_DATA["deployment_type"] == "ipi"
+            and config.ENV_DATA["platform"].lower() == constants.VSPHERE_PLATFORM
+        ):
+            # Create machineset for app worker nodes on vmware
+            ms_name = list()
+            labels = [("node-role.kubernetes.io/app", "app-scale")]
+            for obj in machine.get_machineset_objs():
+                if "app" in obj.name:
+                    ms_name.append(obj.name)
+            if not ms_name:
+                if len(machine.get_machineset_objs()) == 3:
+                    for zone in ["3", "4", "5"]:
```

Comment:
What are the default zones of already existing nodes?  Do we have new zones here? Or existing zone names are 3, 4, 5? Should we somehow dynamically get current zones and add nodes to the existing one dynamically instead of hardcoded numbers here?

Or try to read it from config? As for arbiter deployment we have zones defined in config:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/vsphere/upi_1az_rhcos_vsan_lso_vmdk_3m_6w_arbiter.yaml#L20-L22

But this is UPI, for IPI I don't see we define zones in config (maybe we should).

Reply:
Scale tests are for IPI and yes it makes sense to get the zone name dynamically than hardcoding it, I will make this change.

Reply:
fixed